app/services/checksum_validator.rb

```diff
@@ -28,17 +28,20 @@ def validate_checksum
   end
 
   def validate_manifest_inventories
-    moab_storage_object.version_list.each do |moab_version|
-      validate_manifest_inventory(moab_version)
-    end
+    moab_storage_object.version_list.each { |moab_version| validate_manifest_inventory(moab_version) }
     checksum_results.report_results
   end
 
   def validate_signature_catalog
+    flag_unexpected_data_content_files
+    validate_signature_catalog_listing
+  end
+
+  private
+
+  def validate_signature_catalog_listing
     begin
-      latest_signature_catalog_entries.each do |signature_catalog_entry|
-        validate_signature_catalog_entry(signature_catalog_entry)
-      end
+      latest_signature_catalog_entries.each { |entry| validate_signature_catalog_entry(entry) }
     rescue Errno::ENOENT
       checksum_results.add_result(AuditResults::MANIFEST_NOT_IN_MOAB, manifest_file_path: latest_signature_catalog_path)
```

Comment:
I think this is the wrong error code,no?   It's not a *manifest* that isn't in the Moab, it's the signature catalog file.  We may need to add some more error codes to AuditResults.   Let's not be too stingy with them.